units/units_decl.hpp

```diff
@@ -96,8 +102,14 @@ namespace detail {
         }
         /** Construct with the error flag triggered*/
         explicit constexpr unit_data(std::nullptr_t) :
-            meter_(-8), second_(-8), kilogram_(-4), ampere_(-4), candela_(-2),
-            kelvin_(-4), mole_(-2), radians_(-4), currency_(-2), count_(-2),
+            meter_(maxNeg(bitwidth::meter)), second_(maxNeg(bitwidth::second)),
+            kilogram_(maxNeg(bitwidth::kilogram)),
+            ampere_(maxNeg(bitwidth::ampere)),
+            candela_(maxNeg(bitwidth::candela)),
+            kelvin_(maxNeg(bitwidth::kelvin)), mole_(maxNeg(bitwidth::mole)),
+            radians_(maxNeg(bitwidth::radian)),
+            currency_(maxNeg(bitwidth::currency)),
+            count_(maxNeg(bitwidth::count)),
```

Comment:
Does this mean we should exclude this here: https://github.com/LLNL/units/blob/0df7071438a464742f2486746da0c1c45445a4ab/units/units_util.hpp#L18 for the overflow checks, since it may be an error if all bases happen to be at their minimum? Or is it valid to have one or a few bases at minimum, given that it is extremely unlikely that all of them are?